adafruit_platformdetect/board.py

```diff
@@ -384,7 +388,7 @@ def any_beaglebone(self):
     @property
     def any_orange_pi(self):
         """Check whether the current board is any defined Orange Pi."""
-        return self.ORANGE_PI_PC
+        return self.ORANGE_PI_IDS
```

Comment:
This should return a True/False value instead of a list. Also, the list starts with an underscore character.